ethcore/vm/src/action_params.rs

```diff
@@ -130,7 +130,7 @@ impl From<ethjson::vm::Transaction> for ActionParams {
 			gas: t.gas.into(),
 			gas_price: t.gas_price.into(),
 			value: ActionValue::Transfer(t.value.into()),
-			call_type: match address.is_zero() { true => CallType::None, false => CallType::Call },	// TODO @debris is this correct?
+			action_type: ActionType::Call,
```

Comment:
Doesn't that change the JSON tests results?

Reply:
I haven't touched that code, so I assume it was reviewed
https://github.com/paritytech/parity-ethereum/pull/11311#discussion_r360395162